src/sentry/static/sentry/app/utils/log_exposure.jsx

```diff
@@ -0,0 +1,11 @@
+import HookStore from 'app/stores/hookStore';
+
+/**
+ * @param {String} name name of the experiment
+ * @param {Object} data Experiment data to be recorded: {unit_name, unit_id, params}
+ * 					note that params need to string
+ */
```

Comment:
still not super clear on why passing an object for params doesn't work- will dig in at some point to figure that out.